documentation/src/main/asciidoc/user_guide/cache_api.adoc

```diff
@@ -105,16 +105,17 @@ Events trigger a notification which is dispatched to listeners.   Listeners are
 
 NOTE: Both Cache and CacheManager implement Listenable, which means you can attach listeners to either a cache or a cache manager, to receive either cache-level or cache manager-level notifications.
 
-For example, the following class defines a listener to print out some information every time a new entry is added to the cache:
+For example, the following class defines a listener to print out some information every time a new entry is added to the cache, in a non blocking fashion:
```

Comment:
`println` is not really non-blocking, is it? :)

Reply:
I ofc thought of that, but figured people wouldn't nitpick. But I was obviously wrong :) I can change it to something else.

Reply:
I was actually nitpicking that you're saying "in a non-blocking fashion", but there's nothing in the method that delays the operation without blocking the thread.